catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -0,0 +1,151 @@
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+
+import ddf.catalog.Constants;
+import ddf.catalog.transform.InputTransformer;
+
+/**
+ * Provides information on available transformers
+ */
+@Service
+@Command(scope = CatalogCommands.NAMESPACE, name = "transformers", description = "Provides information on available transformers.")
+public class TransformersCommand extends CatalogCommands {
+
+    // output strings
+
+    static final String ACTIVE_TRANSFORMERS_HEADER = "Active Transformers: ";
+
+    static final String NO_ACTIVE_TRANSFORMERS = "There are no active transformers";
+
+    private static final String LINE = "------------------------";
+
+    // Transformer Properties
+
+    private static final String MIME_TYPE = "mime-type";
+
+    private static final String SCHEMA = "schema";
+
+    private static final String NOT_AVAILABLE = "N/A";
+
+
+
+    @Override
+    protected Object executeWithSubject() throws Exception {
+
+        // ServiceReferences are needed to get property information from InputTransformers
+        List<ServiceReference<InputTransformer>> serviceReferences =
+                (List<ServiceReference<InputTransformer>>) bundleContext.getServiceReferences(
+                        InputTransformer.class,
+                        "(id=*)");
+
+        List<TransformerProperties> transformersProperties = serviceReferences.stream()
+                .map(TransformerProperties::new)
+                .collect(Collectors.toList());
+
+        int activeTransformers = transformersProperties.size();
+
+        if (activeTransformers == 0) {
+            console.printf("%s%n%n", NO_ACTIVE_TRANSFORMERS);
+            return null;
+        }
+
+        console.printf("%n%s%d%n%s%n%n", ACTIVE_TRANSFORMERS_HEADER, activeTransformers, LINE);
+
+        Iterator<TransformerProperties> tpIterator = transformersProperties.iterator();
+        TransformerProperties tp;
+
+        while (tpIterator.hasNext()) {
+            tp = tpIterator.next();
+            console.printf("%s", tp.printProperties());
+
+            if (tpIterator.hasNext()) {
+                console.printf("%n%s%n", StringUtils.repeat(LINE, 4));
```

Comment:
The line is only 96 characters long, so it would take a very narrow terminal to wrap the line. The line helps provide visual assistance to show where a new transformer is getting displayed, so I think its worth the risk. Do you know any terminal that might have a width less than 96 chars?